pandas/tests/frame/test_query_eval.py

```diff
@@ -165,7 +164,10 @@ def test_eval_resolvers_as_list(self):
 
 class TestDataFrameQueryWithMultiIndex(tm.TestCase):
 
-    def check_query_with_named_multiindex(self, parser, engine):
+    # parametrize single parser, for symmetry with other tests
```

Comment:
this is better done with fixtures no? these will then automatically get the cartesian product. now you *may* want to parameterize when you *don't* want that of course (e.g. in this example here you can parameterize on parser, but leave engines as the fixture).

e.g.
```
@pytes.fixture(params=ENGINES)
def engine(request):
    return request.param
```
and same for parser